AssetAdministrationShellRegistryServiceSpecification/V3.1_SSP-005.yaml

```diff
@@ -0,0 +1,103 @@
+openapi: 3.0.3
+info:
+  title: DotAAS Part 2 | HTTP/REST | Registry Service Specification
```

Comment:
only Registry Service or Asset Administration Shell Registry Service since we also have a Submodel Registry service?